sdk/identity/identity/test/manual-integration/AzureArc/INSTRUCTIONS.md

```diff
@@ -0,0 +1,73 @@
+# Testing Identity in Azure Arc
+
+## Prerequisites
+
+- A non-Azure Windows or Linux VM.
+  - Must have NodeJS, NPM, and git installed.
+- Administrator privileges on the VM.
+- An Azure Key Vault.
```

Comment:
Can we link to the Key Vault getting started docs?

Reply:
I think we can be more lenient here, since customers aren't the target audience we can expect that the testing folks know how to set up the proper tooling